views/templates/front/payment.tpl

```diff
@@ -33,12 +43,10 @@
                     let language;
                     let javaEnabled;
 
-                    let adyenCheckout;
-
                     let placeOrderAllowed;
                     let popupModal;
 
-
+                    let storeCc;
```

Comment:
@KadoBOT can you validate if this javascript will work for all common browsers ? I think better to move from let to var.

Reply:
https://caniuse.com/#feat=let
Doesn't work until IE 10